uefi-mkconfig

```diff
@@ -13,152 +13,112 @@ ewarn() {
 	echo -e " ${NOCOLOR-\e[1;33m*\e[0m }${*}" >&2
 }
 
-add_uefi_entries () (
-	for efi_file in $partition_efis; do
-		# Skip this kernel if ignore file is found
-		if [[ -f "$efi_file.ignore" ]]; then
-			ewarn "Ignoring \"$efi_file\" from partition \"$partition\"."
-			continue
-		fi
+add_uefi_entry () (
 
-		# Prepare path which will be inserted into efibootmgr
-		local efi_file_path
-		efi_file_path="${efi_file//$partition_mount}"
+	# Create label for UEFI entry
+	local entry_label
+	entry_label="$kernel_version on $partition_label "
```

Comment:
Is this extra space at the end here intentionally?

Reply:
Yes